### PR TITLE
feat(docker file): Add Dockerfile and docker-compose.yml

### PR DESCRIPTION
**Pull request by spaghettifunk**
* Add Dockerfile to containerize the application
* Add docker-compose.yml file so that it will be easier to add the the bidder-controller and the dsp-api for testing
* Changed build.sbt and assembly.sbt files to be able to package the application in a JAR file

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
 lazy val akkaVersion = "2.5.16"
 
+lazy val root = (project in file(".")).
+  settings(
+    name := "pacer",
+    version := "0.1",
+    organization := "rtl-group",
+    scalaVersion := "2.12.7",
+    parallelExecution in Test := false,
+    mainClass in Compile := Some("myPackage.MyMainObject")
```
Comment (mulya) [inline build.sbt]: mainClass in Compile := Some("nl.rtl.Main")

File: .gitignore [hunk 2]
```diff
@@ -2,4 +2,7 @@
 .idea/hydra.xml
 out/
 
+project/
```
Comment (mulya) [inline .gitignore]: Replace with "project/target". Dir "project" contains file "build.properties" where sbt version specified and it better to be in repo

File: build.sbt (hunk 1 above)
Comment (mulya) [reply build.sbt]: Or remove it at all, I see you specified it lower for "assembly"

File: .gitignore (hunk 2 above)
Comment (spaghettifunk) [reply .gitignore]: will do!

File: build.sbt (hunk 1 above)
Comment (spaghettifunk) [reply build.sbt]: oops! forgot to remove it! do it now

**Pull request merged**